package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "jpd-breader",
-    "version": "10.0",
+    "version": "11.0",
```

Please don't change the version number in pull requests
```suggestion
    "version": "10.0",
```